src/styles/index.scss

```diff
@@ -1,3 +1,82 @@
+* {
+  box-sizing: border-box;
+}
+
 body {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
   margin: 0;
+  font-family: Arial, sans-serif;
+  font-size: 30px;
+}
+
+.calendar__day {
+  background-color: #eee;
+  border: 1px solid #000;
+  justify-content: center;
+  align-items: center;
+  width: 100px;
+  height: 100px;
+  transition:
+    background-color 0.5s,
+    transform 0.5s;
+  display: flex;
+
+  &:hover {
+    cursor: pointer;
+    background-color: #ffbfcb;
+    transform: translateY(-20px);
+  }
+
+  @for $i from 1 through 31 {
+    &:nth-child(#{$i}):before {
+      content: '#{$i}';
+    }
+  }
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+  width: 726px;
+  padding: 10px;
+
+  &--start-day-mon .calendar__day:first-child {
```

Comment:
The `&--start-day-mon .calendar__day:first-child` and similar selectors for other days seem to be repetitive with only the `margin-left` value changing. Consider using a loop to generate these styles or a mixin to reduce repetition and improve maintainability.